lib/faker/bot/commands/list.rb

```diff
@@ -11,6 +11,18 @@ def execute(output: $stdout)
           result = Reflectors::List.call(options)
           render(result, output)
         end
+
+        private
+
+        def render(result, output)
```

Comment:
Nice specialisation here. Eventually, I think we might define the `Command#render` superclass to have this shape since we have the same on the search command. But I think this is great for now as we might add some new commands (don't know which ones yet) and they might have a different shape

Reply:
That’s true, good call. I’ll keep that in mind moving forward!